src/Components/Card.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import "../Styles/Card.css";
+
+export function Card({ cardInfo, key }) {
+  const { imageSource, createdAt, description, url } = cardInfo;
+  const inputDate = new Date(createdAt);
+
+  const year = inputDate.getFullYear();
+  const month = (inputDate.getMonth() + 1).toString().padStart(2, "0");
+  const day = inputDate.getDate().toString().padStart(2, "0");
+  const createdAtDate = `${year}. ${month}. ${day}`;
+
+  const getCreatedFrom = () => {
+    const now = new Date();
+    const timeDifference = now - inputDate;
+
+    const minutes = Math.floor(timeDifference / (1000 * 60));
+    const hours = Math.floor(timeDifference / (1000 * 60 * 60));
+    const days = Math.floor(timeDifference / (1000 * 60 * 60 * 24));
+    const months = Math.floor(timeDifference / (1000 * 60 * 60 * 24 * 30));
+    const years = Math.floor(timeDifference / (1000 * 60 * 60 * 24 * 30 * 12));
+
+    if (minutes < 2) {
+      return "1 minute ago";
+    } else if (minutes <= 59) {
+      return `${minutes} minutes ago`;
+    } else if (hours < 1) {
+      return "1 hour ago";
+    } else if (hours <= 23) {
+      return `${hours} hours ago`;
+    } else if (days < 1) {
+      return "1 day ago";
+    } else if (days <= 30) {
+      return `${days} days ago`;
+    } else if (months < 1) {
+      return "1 month ago";
+    } else if (months <= 11) {
+      return `${months} months ago`;
+    } else if (years < 1) {
+      return "1 year ago";
+    } else {
+      const remainingMonths = months % 12;
+      return `${Math.floor(years)} years ago`;
+    }
+  };
```

Comment:
```suggestion
    if (minutes < 2) {
      return "1 minute ago";
    } 
    if (minutes <= 59) {
      return `${minutes} minutes ago`;
    }
    if (hours < 1) {
      return "1 hour ago";
    }
    if (hours <= 23) {
      return `${hours} hours ago`;
    }
    if (days < 1) {
      return "1 day ago";
    }
    if (days <= 30) {
      return `${days} days ago`;
    }
    if (months < 1) {
      return "1 month ago";
    }
    if (months <= 11) {
      return `${months} months ago`;
    }
    if (years < 1) {
      return "1 year ago";
    }
    const remainingMonths = months % 12;
    return `${Math.floor(years)} years ago`;
  };
```
어차피 if문에서 리턴이 되어서 else if 안써도 될것같아요! 
참고로 앞으로 else if를 사용하시게 된다면 진짜 필요한지 다시 고민해보시는게 좋습니당 ㅋㅋㅋ 대부분의 경우 필요가... 없고 개인적으로 가독성이 좋지않다고 생각해요!